### PR TITLE
i18n(ko): Translate articles in section 'Start Here'

### PR DESCRIPTION
**Pull request by sujang958**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translated `manual.mdx`
- Translated `editor-setup.mdx`
- Translated `v2.mdx`

### CODE REVIEW

File: src/content/docs/ko/editor-setup.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+## 다른 코드 에디터
+
+저희의 멋진 커뮤니티에서 다른 인기있는 에디터들을 위해 다음과 같은 여러가지 확장 기능을 관리하고 있습니다.
+
+- [Open VSX 에 있는 VS Code Extension](https://open-vsx.org/extension/astro-build/astro-vscode) <span style="margin: 0.25em;"><Badge variant="accent">공식</Badge></span> - 공식 Astro VS Code 확장은 [VSCodium](https://vscodium.com/)과 같은 개방형 플랫폼용 Open VSX 레지스트리에서 사용할 수 있습니다.
+- [Nova Extension](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/) <span style="margin: 0.25em;"><Badge variant="neutral">커뮤니티</Badge></span> - 노바 내에서 Astro에 구문 강조 표시 및 코드 완성 기능을 제공합니다.
+- [Vim Plugin](https://github.com/wuelnerdotexe/vim-astro) <span style="margin: 0.25em;"><Badge variant="neutral">커뮤니티</Badge></span> - Vim 또는 Neovim 내에서 Astro에 구문 강조 표시, 들여쓰기 및 코드 접기 지원을 제공합니다.
+- Neovim [LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#astro) and [TreeSitter](https://github.com/virchau13/tree-sitter-astro) Plugins <span style="margin: 0.25em;"><Badge variant="neutral">커뮤니티</Badge></span> - Neovim 내에서 Astro를 위한 구문 강조 표시, 트리시터 구문 분석 및 코드 완성 기능을 제공합니다.
+
+## 브라우저 안 에디터
```
Comment (filename24) [inline src/content/docs/ko/editor-setup.mdx:42]:
```suggestion
## 웹 에디터
```

이게 더 직관적이고 읽을떼 편한거같습니다.

File: src/content/docs/ko/guides/upgrade-to/v2.mdx [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+
+### 변경됨: `Astro.site`의 후행 슬래시
+
+Astro v1.x에서 Astro는 `Astro.site`를 사용하여 액세스할 때 `astro.config.mjs`에서 `site`로 설정한 URL에 항상 후행 슬래시가 있는지 확인했습니다.
+
+Astro v2.0은 더 이상 `site`의 값을 수정하지 않습니다. `Astro.site`는 정의되어있는 정확한 값을 사용하며, 원하는 경우 후행 슬래시를 추가해야합니다.
+
+#### 무엇을 해야 하나요?
+
+ astro.config.mjs`에서 `site`에 설정된 URL에 후행 슬래시를 추가합니다.
```
Comment (filename24) [inline src/content/docs/ko/guides/upgrade-to/v2.mdx:94]: 여기 코드블럭 오류가 있네요.

**Review by filename24 — CHANGES_REQUESTED** [review]

File: src/content/docs/ko/editor-setup.mdx (hunk 1 above)
Comment (sujang958) [reply src/content/docs/ko/editor-setup.mdx:42]: 그런 것 같네요, 수정하겠습니다!

File: src/content/docs/ko/guides/upgrade-to/v2.mdx (hunk 2 above)
Comment (sujang958) [reply src/content/docs/ko/guides/upgrade-to/v2.mdx:94]: 수정하겠습니다!

**Review by filename24 — APPROVED** [review]

File: src/content/docs/ko/guides/upgrade-to/v2.mdx [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+
+### 업데이트: Vite 4
+
+Astro v2.0는 Vite 3에서 2022년 10월에 출시된 [Vite 4](https://vitejs.dev/)으로 업그레이드 했습니다.
+
+#### 무엇을 해야 하나요?
+
+이번에는 코드에 변경할께 없습니다! Astro 내부에서 대부분의 업그레이드를 처리했지만, 일부 세부적인 Vite 동작은 버전 간에 변경점이 있을수있습니다.
+
+문제가 생기면 [공식 Vite 마이그레이션 가이드](https://vitejs.dev/guide/migration.html)을 참조하세요
```
Comment (imjlk) [inline src/content/docs/ko/guides/upgrade-to/v2.mdx:392]:
```suggestion
문제가 생기면 [공식 Vite 마이그레이션 가이드](https://vitejs.dev/guide/migration.html)를 참조하세요
```

File: src/content/docs/ko/guides/upgrade-to/v2.mdx [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+
+### 삭제됨: `Astro.fetchContent()`
+
+v0.26에서 Astro는 로컬 마크다운 파일에서 데이터를 가져오기 위한 `Astro.fetchContent()`를 더 이상 지원하지 않습니다.
+
+Astro v2.0에서는 이 옵션이 완전히 제거되었습니다. 코드에서 `Astro.fetchContent()`를 사용하면 오류가 발생합니다.
+
+#### 무엇을 해야 하나요?
+
+마크다운 파일을 가져오기 위해 [`Astro.glob()`](/ko/guides/imports/#astroglob) 을 사용하거나 [Content Collections](/ko/guides/content-collections/) 기능으로 바꾸세요.
```
Comment (imjlk) [inline src/content/docs/ko/guides/upgrade-to/v2.mdx:357]:
```suggestion
마크다운 파일을 가져오기 위해 [`Astro.glob()`](/ko/guides/imports/#astroglob)을 사용하거나 [Content Collections](/ko/guides/content-collections/) 기능으로 바꾸세요.
```

File: src/content/docs/ko/install/manual.mdx [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+#### 기본 요구 사항
+
+- **Node.js** - `v16.12.0` 또는 상위 버전.
+- **텍스트 에디터** - [VS Code](https://code.visualstudio.com/)와 [공식 Astro 확장 프로그램](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)을 함께 사용하는 것을 추천해요.
+- **터미널** - Astro를 제어할 명령줄 인터페이스(CLI)가 필요합니다.
+
+#### 설치하기
+
+Astro 프로젝트를 자동으로 생성하는 'create astro' CLI 도구를 사용하지 않으려면 아래 가이드에 따라 프로젝트를 직접 설정할 수 있습니다.
```
Comment (imjlk) [inline src/content/docs/ko/install/manual.mdx:25]:
```suggestion
Astro 프로젝트를 자동으로 생성하는 `create astro` CLI 도구를 사용하지 않으려면 아래 가이드에 따라 프로젝트를 직접 설정할 수 있습니다.
```

**Comment by sujang958** [comment]
자세한 리뷰 감사합니다!

**Review by imjlk — CHANGES_REQUESTED** [review]
띄어쓰기 부분 몇 군데만 더 추가했습니다!

**Review by imjlk — APPROVED** [review]

File: src/content/docs/ko/guides/upgrade-to/v2.mdx [hunk 6]
```diff
@@ -0,0 +1,420 @@
...
+// astro.config.mjs
+export default defineConfig({
+	legacy: {
+		astroFlavoredMarkdown: true,
+	},
+})
+
+```
+
+만약 v1.x에서 이 기능을 사용 중이라면 컴포넌트와 JSX 표현식을 마크다운 구문과 결합할 수 있는 [MDX 통합](/ko/guides/통합-가이드/mdx/)을 사용하는 것이 좋습니다.
```
Comment (Yan-Thomas) [inline src/content/docs/ko/guides/upgrade-to/v2.mdx:326]:
```suggestion
만약 v1.x에서 이 기능을 사용 중이라면 컴포넌트와 JSX 표현식을 마크다운 구문과 결합할 수 있는 [MDX 통합](/ko/guides/integrations-guide/mdx/)을 사용하는 것이 좋습니다.
```

File: src/content/docs/ko/install/manual.mdx [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+  - env.d.ts
+- astro.config.mjs
+- package-lock.json or `yarn.lock`, `pnpm-lock.yaml`, etc.
+- package.json
+- tsconfig.json
+</FileTree>
+
+축하합니다, 이제 Astro를 사용할 준비가 되었습니다!
+
+이 가이드를 완전히 따랐다면 [2단계: Astro 시작하기](/ko/install/auto/#2-start-astro-)로 바로 이동하여 Astro를 처음 실행하는 방법을 계속 학습할 수 있습니다.
```
Comment (Yan-Thomas) [inline src/content/docs/ko/install/manual.mdx:194]:
```suggestion
이 가이드를 완전히 따랐다면 [2단계: Astro 시작하기](/ko/install/auto/#2-astro-시작하기-)로 바로 이동하여 Astro를 처음 실행하는 방법을 계속 학습할 수 있습니다.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @sujang958 for the PR, also thanks @filename24 and @imjlk for the careful review. I went ahead and fixed the couple broken links in the page, so then we're good to merge! 🥳

**Pull request merged**